src/m365/tenant/commands/service/service-message-list.ts

```diff
@@ -0,0 +1,96 @@
+import commands from '../../commands';
+import request from '../../../../request';
+import GlobalOptions from '../../../../GlobalOptions';
+import { CommandOption } from '../../../../Command';
+import SpoCommand from '../../../base/SpoCommand';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  workload: string;
+}
+
+class TenantServiceMessageListCommand extends SpoCommand {
+  public get name(): string {
+    return `${commands.TENANT_SERVICE_MESSAGE_LIST}`;
+  }
+
+  public get description(): string {
+    return 'Gets the service messages regarding services in Office 365 from the Office 365 Management API';
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: (err?: any) => void): void {
+    if (this.verbose) {
+      cmd.log(`Getting the service messages regarding services in Office 365 from the Office 365 Management API`);
+    }
+
+    const serviceUrl: string = 'https://manage.office.com/api/v1.0';
+    const statusEndpoint: string = (typeof args.options.workload !='undefined' && args.options.workload) ? `ServiceComms/Messages?$filter=Workload eq '${args.options.workload}'` : 'ServiceComms/Messages';
```

Comment:
Since `workload` is provided by users, we should escape it before passing it to the query